### PR TITLE
Feature/recurring comments

### PR DESCRIPTION
**Pull request by SwethaJanardhana**

### CODE REVIEW

File: components/Form.js [hunk 1]
```diff
@@ -281,21 +290,8 @@ export default function Form({
...
-            isEdit &&
-            (value?.repeat.includes("monthly") ||
-              value?.repeat.includes("weekly") ||
-              value?.repeat.includes("daily")) &&
-            lastDay
-          }
+          min={isEdit ? minDate : formattedTodayDate}
+          max={isEdit && maxDate}
           defaultValue={value?.dueDate || formattedTodayDate}
         ></StyledDateInput>
```
Comment (F-Kirchhoff) [inline components/Form.js:296]: please use self closing tags whenever possible

File: components/Form.js [hunk 2]
```diff
@@ -329,6 +326,25 @@ export default function Form({
             ></StyledDateInput>
```
Comment (F-Kirchhoff) [inline components/Form.js:326]: here as well. please check your entire codebase and update it accordingly

File: pages/api/tasks/[id].js [hunk 3]
```diff
@@ -26,77 +26,56 @@ export default async function handler(request, response) {
...
               groupId: updatedTask.groupId,
-              dueDate: { $gte: updatedTask.dueDate },
               isDone: { $ne: true },
+            }).sort({
+              dueDate: 1,
             })
-          : await Task.deleteMany({
+          : await Task.find({
               groupId: updatedTask.groupId,
               isDone: { $ne: true },
```
Comment (F-Kirchhoff) [inline pages/api/tasks/[id].js:39]: couldn't you pass here isDone: false?

**Review by F-Kirchhoff — APPROVED** [review]
LGTM, please only update the component calls to be self closing if possible.

**Comment by SwethaJanardhana** [comment]
Thank you so much for the review, implement already self-closing tags

**Pull request merged**